docs/source/getstarted.rst

```diff
@@ -59,101 +38,55 @@ Install LeApp-To using these steps: ::
-
-The prototype requires Vagrant with some relevant plugins: ::
+    
+Enable and Start  Docker&Cockpit
+---------------------------
 
-    sudo vagrant plugin install ansible hitimes nio4r vagrant-libvirt
+    systemctl enable docker
+    systemctl enable cockpit
+    systemctl start docker
+    systemctl start cockpit
 
-And passwordless access to VM management operations: ::
-
-    sudo usermod -aG vagrant,libvirt $USER
-
-You will need to log out and back in again, or start a new user
-session with `su $USER`, to get the new group memberships to take
-effect.
-
-Finally, start all the demonstration VMs by running: ::
-
-    ln -s ../available/centos6-guest demo/vmdefs/enabled/
-    ln -s ../available/centos7-target demo/vmdefs/enabled/
-    sudo demo/vmctl.sh provision
-
-This script iterates over all the subdirectories of `demo/vmdefs/enabled` and runs
-`vagrant up --provision`.
-
-Note that the VMs shall be started by the same user who will run the
-demo, i.e. when using sudo to run the demo, one shall use sudo to
-start the VMs as well.
 
 Running the demonstration via the CLI
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-First, check that the source VM is showing the
-PHP admin login page at port 9000, while the target VM isn't
-running a HTTP server.
-
-The demo admin login credentials are:
-
 * Username: `root`
 * Password: `toor`
 
-Then, from the base of the local clone, run: ::
-
-    sudo bin/leapp-tool migrate-machine \
-        --target-user vagrant \
-        --target-identity integration-tests/config/leappto_testing_key \
-        --source-user vagrant \
-        --source-identity integration-tests/config/leappto_testing_key \
-        --tcp-port 9000:9000 -t centos7-target centos6-app-vm
+Then, from the base of the local clone, run::
 
-The target VM should now be showing the PHP admin page,
-with the same information as the source VM.
+    /usr/bin/leapp-tool migrate-machine --use-rsync \
+    --ignore-default-port-map \
+    -t 127.0.0.1 source.myapp.com \
+    --force-createa 
 
+Note: replace source.myapp.com with IP/FQDN of your VM
+Note: make sure you can access from CLI your source VM as user root with an SSH key added to ssh-agent
 
 Running the demonstration via Cockpit
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Link the Cockpit plugin (if not already linked): ::
-
-    mkdir -p ~/.local/share/cockpit
-    ln -snf $PWD/cockpit ~/.local/share/cockpit/leapp
-
-Link the `leapp` project directory (if not already linked): ::
-
-    sudo ln -snf $PWD /opt/leapp
+[ YOUTUBE EMBED HERE ]
 
 Open Cockpit in your browser:
 
     http://localhost:9090
 
-When authenticating, check the option to allow Cockpit to retain your password for later
-privilege escalation. Open **Tools->Le-App** from the navigation menu. Then check that the 
-application link for the source VM show the PHP info page, while the target VM isn't
-running a HTTP server.
+* Access cockpit plugin 
+  Tools -> Import Apps
```

Comment:
This would be clearer as something like "Select 'Tools -> Import Apps' in the navigation sidebar"